config/config.js

```diff
@@ -28,8 +28,32 @@ const config = {
         database: 'test'
     },
 
+    VALIDATOR: {
+        /**
+         * Add custom validators here
+         * Format: { "validatorName": function(value, [additional arguments]), ... }
+         */
+        customValidators: {
```

Comment:
well I added the custom validators that we mostly use... Although this is better than what we've had before since we can actually check if the values are really valid not only the type of the values which `util.get_data` performs before.